web-server/pages/api/internal/ai/dora_metrics.ts

```diff
@@ -125,14 +125,12 @@ endpoint.handle.POST(postSchema, async (req, res) => {
     });
   }
 });
-
-const checkForErrors = (responses: any): { status: string; message: string } => {
+const checkForErrors = (responses: { [key: string]: { status: string; message: string } }): { status: string; message: string } => {
```

Comment:
use `Record`, not that what you have written is wrong, but consistent throughout the codebase

Reply:
Sure